app/templates/DonationForm.html.twig

```diff
@@ -142,6 +163,27 @@
                     {
                         viewHandler: WMDE.View.createSlidingVisibilitySwitcher( $( '.personal-data-full' ), /firma|person/ ),
                         stateKey: 'formContent.addressType'
+                    },
+                    {
+                        viewHandler: WMDE.View.createValueDependentEventHandler( $( '#continueFormSubmit1' ), 'click', [
```

Comment:
What's the deal with this 1 and 2 stuff? The first and second steps on the client?